packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1890,6 +1941,14 @@ export class ContainerRuntime
 				case ContainerMessageType.BlobAttach:
 					this.blobManager.processBlobAttachOp(message, local);
 					break;
+				case ContainerMessageType.IdAllocation:
+					if (this.runtimeOptions.enableRuntimeIdCompressor !== true) {
```

Comment:
Throwing on this.idCompressor === undefined might be better else it can again fail when referencing idCompressor.